### PR TITLE
fix(curriculum): change descriptions of id from properties to attributes

### LINKED ISSUE #47423: Wrong word in step 17, 28, 37 of Building a registration form of RWD
### Describe the Issue

The text refers to `id`s as properties instead of attributes. 

### Affected Page

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff919a7b5612c0670923a5.md, https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8b9dab5ac88e4d3d43a3.md, https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8e998d3e7eae14d6ae3b.md

### Your code

https://github.com/freeCodeCamp/freeCodeCamp/blob/aa4404e05af522af8abb5c83f3e145bb6febee4b/curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8b9dab5ac88e4d3d43a3.md?plain=1#L12

https://github.com/freeCodeCamp/freeCodeCamp/blob/aa4404e05af522af8abb5c83f3e145bb6febee4b/curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8e998d3e7eae14d6ae3b.md?plain=1#L12

https://github.com/freeCodeCamp/freeCodeCamp/blob/aa4404e05af522af8abb5c83f3e145bb6febee4b/curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff919a7b5612c0670923a5.md?plain=1#L12

### Expected behavior

The text should use the term "attribute"

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by adameshg**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47423

The text in 3 challenges in the Responsive Web Design curriculum incorrectly referred to `ids` as "properties" rather than "attributes". This PR fixes this error :)

### CODE REVIEW

**Review by Sboonny — COMMENTED** [review]
What do you think of Shaun [comment](https://github.com/freeCodeCamp/freeCodeCamp/issues/47423#issuecomment-1233271399)?

His suggestion is helpful for translation.

**Comment by adameshg** [comment]
> What do you think of Shaun [comment](https://github.com/freeCodeCamp/freeCodeCamp/issues/47423#issuecomment-1233271399)?
> 
> His suggestion is helpful for translation.

Ah, my bad, I'd missed that -- thank you for pointing it out! To make sure I'm understanding correctly, should the values in all 3 examples be `first-name`, `last-name`, `email`, `new-password`?

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8b9dab5ac88e4d3d43a3.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ dashedName: step-17
 
 Following accessibility best practices, link the `input` elements and the `label` elements together using the `for` attribute.
 
-Use these values for the respective `id` properties: `first-name`, `last-name`, `email`, `new-password`
+Use these values for the respective `id` attributes: `first-name`, `last-name`, `email`, `new-password`
```
Comment (Sboonny) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8b9dab5ac88e4d3d43a3.md:12]:
```suggestion
Use `first-name`, `last-name`, `email`,  and `new-password` as values for the respective `id` attributes.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8e998d3e7eae14d6ae3b.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ dashedName: step-28
 
 Follow accessibility best practices by linking the `input` elements and the `label` elements in the second `fieldset`.
 
-Use these values for the respective `id` properties: `personal-account`, `business-account`, `terms-and-conditions`
+Use these values for the respective `id` attributes: `personal-account`, `business-account`, `terms-and-conditions`
```
Comment (Sboonny) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8e998d3e7eae14d6ae3b.md:12]:
```suggestion
Use `personal-account`, `business-account`, and `terms-and-conditions` as values for the respective `id` attributes.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff919a7b5612c0670923a5.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ dashedName: step-37
 
 Link the applicable form elements and their `label` elements together.
 
-Use these values for the respective `id` properties: `profile-picture`, `age`, `referrer`, `bio`
+Use these values for the respective `id` attributes: `profile-picture`, `age`, `referrer`, `bio`
```
Comment (Sboonny) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff919a7b5612c0670923a5.md:12]:
```suggestion
Use `profile-picture`, `age`, `referrer`, and `bio` as values for the respective `id` attributes.
```

**Review by Sboonny — CHANGES_REQUESTED** [review]
No worries, I thought we missed something out.

> Ah, my bad, I'd missed that -- thank you for pointing it out! To make sure I'm understanding correctly, should the values in all 3 examples be `first-name`, `last-name`, `email`, `new-password`?

Not really, the values don't need to be altered. We can change the arrangement only

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @adameshg, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Comment by SaadAnsari17** [comment]
> Checklist:
> 
> * [x]  I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
> * [x]  My pull request has a descriptive title (not a vague title like `Update index.md`)
> * [x]  My pull request targets the `main` branch of freeCodeCamp.
> * [x]  I have tested these changes either locally on my machine, or GitPod.
> 
> Closes #47423
> 
> The text in 3 challenges in the Responsive Web Design curriculum incorrectly referred to `ids` as "properties" rather than "attributes". This PR fixes this error :)

Kindly check if a pr is already made before making your own pr.

**Comment by Sboonny** [comment]
@SaadAnsari17 they have not broken our guidelines, so there is no warranty for that warning/request.

**Review by jeremylt — APPROVED** [review]

**Pull request merged**